package.json

```diff
@@ -80,7 +80,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": [
+    "*.styled.{js,jsx,ts,tsx}": [
```

Comment:
Right now, only styled files will be validated. This is not correct behaviour.

We want to do something like that:
`*.{js,jsx,ts,tsx} except styled files` => prettier, eslint
`*.styled.{js,jsx,ts,tsx}` => prettier, eslint, csslint